src/main/java/commands/CommandType.java

```diff
@@ -0,0 +1,23 @@
+package commands;
+
+public enum CommandType {
+    LIST("list"),
+    MARK("mark"),
+    UNMARK("unmark"),
+    DELETE("delete"),
+    TODO("todo"),
+    DEADLINE("deadline"),
+    EVENT("event"),
+    UNKNOWN("unknown"),
+    BYE("bye");
+
+    private final String value;
+
+    CommandType(String value) {
+        this.value = value;
+    }
+
+    public boolean equals(String input) {
```

Comment:
Perhaps the methods should be called `isEqual`?

Reply:
I think `isEqual` is a good idea!